filters/ReprojectionFilter.cpp

```diff
@@ -91,7 +93,39 @@ void ReprojectionFilter::createTransform(const SpatialReference& srsSRS)
             throwError("source data has no spatial reference and "
                 "none is specified with the 'in_srs' option.");
     }
-    m_transform.reset(new SrsTransform(m_inSRS, m_outSRS));
+
+    if (m_inAxisOrdering.size() || m_outAxisOrdering.size())
```

Comment:
This can be eliminated by using a decent default.

Reply:
This should be done in `prepared()` so that processing can be stopped during argument parsing.  Also, it would be good to check that the vector and values are reasonable and provide some feedback.  You should only ever have vector sizes of 2 or 3 and they should always be the values 1,2 or 1,2,3 when sorted.

Reply:
> a decent default.

I don't know what this means. The default is managed by GDAL, and any setting of it potentially triggers something to happen internally with the OSR class. 

I'll implement this in `prepared()`.

Reply:
See latest. I added a member to SpatialReference to getAxisOrdering, which forwards the ordering from OSR to provide a default if needed, but it is not used.

Reply:
👍